src/components/Submission.js

```diff
@@ -38,48 +59,60 @@ const Column = styled.div`
 
 const FeedbackCard = ({ feedback }) => {
   return (
-    <Card>
-      {feedback.map(item => (
-        <FeedbackItem key={item}>{item}</FeedbackItem>
-      ))}
-    </Card>
+    <>
+      {feedback.map((item, i) => {
+        const ordered = {}
+        Object.keys(item)
+          .sort()
+          .forEach(function (key) {
+            ordered[key] = item[key]
+          })
+
+        const grades = Object.entries(ordered).filter(([key, value]) => key !== 'notes')
+        const total = Object.values(grades).reduce((accum, cur) => accum + cur[1], 0)
```

Comment:
not a blocker but would prefer this to be done in a container not in the component